modules/bluefish/bluefish.cpp

```diff
@@ -78,21 +75,42 @@ std::vector<std::wstring> device_list()
 
 void init(core::module_dependencies dependencies)
 {
+
+	bool bCanBeConsumer = false;
```

Comment:
Please don't use camel case (only lower case with underscores) and also don't use hungarian notation.